### ISSUE TITLE
Bug report: The evaluation function of GAT on PPI dataset

### ISSUE THREAD

**Issue by ycjing** [dmlc/dgl#1532]
## To Reproduce

1. Run train_ppi.py (https://github.com/dmlc/dgl/blob/master/examples/pytorch/gat/train_ppi.py)

```
def evaluate(feats, model, subgraph, labels, loss_fcn):
    with torch.no_grad():
        model.eval()
        model.g = subgraph
        for layer in model.gat_layers:
            layer.g = subgraph
        output = model(feats.float())
        loss_data = loss_fcn(output, labels.float())
        predict = np.where(output.data.cpu().numpy() >= 0.5, 1, 0)
        score = f1_score(labels.data.cpu().numpy(),
                         predict, average='micro')
        return score, loss_data.item()
```

## Expected behavior

I think that the line “predict = np.where(output.data.cpu().numpy() >= 0.5, 1, 0)” should be “predict = np.where(output.data.cpu().numpy() >= 0.0, 1, 0)”, since the output variable is directly from the GAT output layer without the sigmoid function.

## Environment

 - DGL Version (e.g., 1.0): 0.4.2
 - Backend Library & Version (e.g., PyTorch 0.4.1, MXNet/Gluon 1.3): PyTorch 1.4
 - OS (e.g., Linux): Linux
 - How you installed DGL (`conda`, `pip`, source): conda
 - Build command you used (if compiling from source): 
 - Python version: 3.6
 - CUDA/cuDNN version (if applicable):
 - GPU models and configuration (e.g. V100): TITAN V
 - Any other relevant information:

## Additional context

None

**Comment by ycjing** [comment]
Already got the response from https://discuss.dgl.ai/t/bug-reported-the-evaluation-function-of-gat-ppi/979/3. Thank you.
